platform-definition.yaml

```diff
@@ -435,6 +440,9 @@ platform_definition:
       'org.springframework.data:spring-data-gemfire':
          - 'org.springframework.shell:spring-shell'
          - 'org.iq80.snappy:snappy'
+      'org.springframework.data:spring-data-geode':
+         - 'org.springframework.shell:spring-shell'
+         - 'org.iq80.snappy:snappy'
```

Comment:
Can you confirm that this is a test dependency in 2.0.3 snapshots? That should now be the case for SD GemFire as a result of [SGF-701](https://jira.spring.io/browse/SGF-701).

Reply:
Yes, that is correct.  Good catch; thank you.

Reply:
Yes; `io.iq80.snappy:snappy` is a "**test**" dependency in both [_Spring Data GemFire_](https://github.com/spring-projects/spring-data-gemfire/blob/2.0.x/pom.xml#L215-L220), [2.0.3.SNAPSHOT](https://github.com/spring-projects/spring-data-gemfire/blob/2.0.x/pom.xml#L14) as well as for [_Spring Data Geode_](https://github.com/spring-projects/spring-data-geode/blob/2.0.x/pom.xml#L238-L243), [2.0.3.BUILD-SNAPSHOT]https://github.com/spring-projects/spring-data-geode/blob/2.0.x/pom.xml#L14)))